docs/index.rst

```diff
@@ -30,6 +30,7 @@ Welcome to DeepPavlov's documentation!
    Context Question Answering <features/models/squad>
    Classification <features/models/classifiers>
    Entity Extraction <features/models/entity_extraction>
+   Few-shot Classification <features/models/few_shot_intent_classification>
```

Comment:
Should few-shot classification be separated section? Maybe it could be described at Classification.
@vaskonov

Reply:
I personally prefer a separated section for few-shot classification. In this case it's more noticeable for users.

Reply:
In this case, @LogicZMaksimka place it after Classification, not Entity Extraction